docs/src/config/ini-config.adoc

```diff
@@ -461,7 +472,9 @@ http://wiki.linuxcnc.org/[http://wiki.linuxcnc.org/].
 If the environment variable AXIS_PROGRESS_BAR is set, then lines
 written to stderr of the form
 
-* 'FILTER_PROGRESS=%d'
+----
+FILTER_PROGRESS=%d
+----
```

Comment:
same here

Reply:
OK, converted this one to list item too and reordered a couple of paragraphs to group Python related ones before this... Looks better indeed :+1: